helm/eirini/templates/eirini-loggregator-bridge.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: eirini-logging-extension
+  labels:
+    app: eirini-logging-extension
+spec:
+  serviceAccountName: "opi"
+  volumes:
+  - name: config
+  - name: cf-secrets
+    secret:
+      secretName: secrets-{{ .Values.scf.version }}-{{ .Values.scf.secrets_generation_counter }}
+      items:
+      - key: doppler-cert
+        path: doppler.crt
+      - key: doppler-cert-key
+        path: doppler.key
+      - key: internal-ca-cert
+        path: doppler.ca
+  initContainers:
+  - name: create-eirini-loggregator-bridge-config
+    volumeMounts:
+    - name: config
+      mountPath: /config
+    image: splatform/eirini-loggregator-bridge
```

Comment:
https://github.com/SUSE/eirini-loggregator-bridge/blob/master/Dockerfile . This will need to be built in some pipeline.